client/jetpack-onboarding/steps/summary.jsx

```diff
@@ -19,7 +21,17 @@ class JetpackOnboardingSummaryStep extends React.PureComponent {
 			'You enabled Jetpack and unlocked dozens of website-bolstering features. Continue preparing your site below.'
 		);
 
-		return <FormattedHeader headerText={ headerText } subHeaderText={ subHeaderText } />;
+		const siteRedirectHref = '/jetpack/onboarding/site-title';
+
+		return (
+			<div className="steps__summary">
```

Comment:
If we remove all the CSS, we probably won't need this `className` for now, which means you can replace the `div` with a `React.Fragment` just like the other steps.